Cargo.toml

```diff
@@ -4,4 +4,5 @@ members = [
     "tonic-build",
     "tonic-examples",
     "tonic-interop",
+    "tonic-benches",
```

Comment:
I want to see if its possible to include these within the main tonic crate? We can avoid the build.rs if we just hand write what the codegen does. What do you think?

Reply:
@LucioFranco It simplifies everything.  

I've updated the PR here for review, pushing the criterion tests inside tonic with no proto build.    There are some comments in a readme.  There is a new dev-dep in the cargo.toml

If it makes sense, I'll open a clean PR b/c I obviously still don't understand everything about git. ;)